scripts/completion/dvc.zsh

```diff
@@ -216,7 +216,7 @@ _dvc_repro=(
   {-R,--recursive}"[Reproduce all stages in the specified directory.]"
   "--ignore-build-cache[Reproduce all descendants of a changed stage even if their direct dependencies didn't change.]"
   "--no-commit[Don't put files/directories into cache.]"
-  "--downstream[Reproduce the pipeline starting from the specified stage.]"
+  "--downstream[Reproduce pipeline(s) starting from the specified stage.]"
```

Comment:
if take a single stage, then it's about reproducing a single pipeline only. If it accepts multiple targets from different pipelines (I would expect so), then we need to change the "specified stage" part here

Reply:
True! Should be `stage(s)` indeed. Let me double check and will update.